addons/account/wizard/accrued_orders.py

```diff
@@ -52,7 +51,7 @@ def _get_default_company(self):
         check_company=True,
         domain=_get_account_domain,
     )
-    preview_data = fields.Binary(compute='_compute_preview_data')
+    preview_data = fields.Text(compute='_compute_preview_data')
```

Comment:
It would actually be better to have only Binary fields to avoid encoding/decoding the Json :shrug: